### PR TITLE
Add bower.json

### PR DESCRIPTION
**Pull request by nathanboktae**
I see you're already registered on `bower`, but this will remove some warnings and helps discoverability of cherrytree overall.

### CODE REVIEW

**Comment by KidkArolis** [comment]
I'm kind of strongly against bower. I think there's nothing bower can do that npm can't. So I'd prefer to not maintain support for bower which is why I've removed bower.json in the past.

Hm..

**Comment by nathanboktae** [comment]
You don't have to maintain it. It's a one-time thing. Moreover, do you really want your opinion on npm vs bower to affect adoption of cherrytree? I'd hope not.

I'd rather not turn this into a bower vs npm argument, but if it has too - two points bower has over npm for front-end:

1. Front-end cares about payload size, so having only one copy of a library is highly desired. Bower has let me overwrite ranges explicitly which has been a huge help. Sure `npm link` exists, but I haven't [heard good things about it](https://twitter.com/thomasABoyt/status/514800651608784896).
2. `main` accepts an [array of items](https://github.com/bower/spec/blob/master/json.md), as with the browser there is more than just javascript, but CSS, templates, and images, some required and some not.

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cherrytree",
+  "description": "Cherrytree - a flexible hierarchical client side router",
+  "main": "standalone.js",
```
Comment (KidkArolis) [inline bower.json]: This should probably be `build/standalone.js`?

Comment (KidkArolis) [inline bower.json]: I'll merge and update on master

**Comment by nathanboktae** [comment]
Thanks!

**Pull request merged**